idunn/utils/default_settings.yaml

```diff
@@ -5,6 +5,6 @@ WIKI_USER_AGENT: 'Idunn' # Used in requests to external wiki* APIs
 
 DEFAULT_LANGUAGE: 'en' # Fallback when no 'lang' in request
 
-CIRCUIT_TIMEOUT: 120
```

Comment:
TIMEOUT seems way too generic.

I think we even need to be more specific and put `WIKI_API_CIRCUIT_TIMEOUT` ?

Reply:
done